src/Sulu/Bundle/MediaBundle/Media/Manager/MediaManager.php

```diff
@@ -351,6 +370,8 @@ private function modifyMedia($uploadedFile, $data, $user)
             throw new FileVersionNotFoundException($mediaEntity->getId(), $version);
         }
 
+        $shouldThrowModifiedEvent = true;
```

Comment:
```suggestion
        $shouldOmitModifiedEvent = true;
```

Reply:
I guess you meant `emit` 🙂

Reply:
you are right 😅  `emit` or `dispatch`. `throw` sounds a bit like an exception